utils/rpms/daos.spec

```diff
@@ -0,0 +1,227 @@
+Summary:       DAOS Storage Engine
+
+License:       Apache
+URL:           https//github.com/daos-stack/daos
+Source0:       %{name}-%{version}.tar.gz
+Source1:       scons_local-%{version}.tar.gz
+
+BuildRequires: scons
+BuildRequires: cart-devel
+BuildRequires: argobots-devel >= 1.0rc1
+BuildRequires: libpmem-devel, libpmemobj-devel
+BuildRequires: fuse-devel >= 3.4.2
+BuildRequires: protobuf-c-devel
+BuildRequires: spdk-devel, spdk-tools
+BuildRequires: fio < 3.4
+BuildRequires: libisa-l-devel
+BuildRequires: raft-devel
+BuildRequires: mercury-devel
+BuildRequires: openpa-devel
+BuildRequires: libfabric-devel
+BuildRequires: openssl-devel
+BuildRequires: ompi-devel
+BuildRequires: pmix-devel
+BuildRequires: hwloc-devel
+BuildRequires: libevent-devel
+BuildRequires: libyaml-devel
+BuildRequires: libcmocka-devel
+BuildRequires: readline-devel
+%if (0%{?rhel} >= 7)
+BuildRequires:  numactl-devel
+BuildRequires: CUnit-devel
+BuildRequires: golang-bin
+BuildRequires: libipmctl-devel
+%else
+%if (0%{?suse_version} >= 1315)
+BuildRequires:  libnuma-devel
+BuildRequires: cunit-devel
+BuildRequires: go1.10
+BuildRequires: ipmctl-devel
+%endif
+%endif
+Requires: cart
+Requires: argobots >= 1.0rc1
+Requires: libpmem, libpmemobj
+Requires: fuse >= 3.4.2
+Requires: protobuf-c
+Requires: spdk
+Requires: fio < 3.4
+
+%description
+The Distributed Asynchronous Object Storage (DAOS) is an open-source
+software-defined object store designed from the ground up for
+massively distributed Non Volatile Memory (NVM). DAOS takes advantage
+of next generation NVM technology like Storage Class Memory (SCM) and
+NVM express (NVMe) while presenting a key-value storage interface and
+providing features such as transactional non-blocking I/O, advanced
+data protection with self healing on top of commodity hardware, end-
+to-end data integrity, fine grained data control and elastic storage
+to optimize performance and cost.
+
+%package server
+Summary: The DAOS server
+Requires: %{name} = %{version}-%{release}
+Requires(post): /sbin/ldconfig
+Requires(postun): /sbin/ldconfig
+
+%description server
+This is the package needed to run a DAOS server
+
+%package client
+Summary: The DAOS client
+Requires: %{name} = %{version}-%{release}
+
+%description client
+This is the package needed to run a DAOS client
+
+%package tests
+Summary: The DAOS test suite
+Requires: %{name}-client = %{version}-%{release}
+Requires: python-pathlib
+
+%description tests
+This is the package needed to run the DAOS test suite
+
+%package devel
+Summary: The DAOS development libraries and headers
+
+%description devel
+This is the package needed to build software with the DAOS library.
+
+%prep
+%setup -q
+%setup -q -a 1
+
+
+%build
+# remove rpathing from the build
+rpath_files="utils/daos_build.py"
+rpath_files+=" $(find . -name SConscript)"
+sed -i -e '/AppendUnique(RPATH=.*)/d' $rpath_files
+
+scons %{?no_smp_mflags}    \
+      --config=force       \
+      USE_INSTALLED=all    \
```

Comment:
do we definitely want to use installed?

Reply:
We definitely do.  We want to use all of the components/dependencies that we have built and installed as RPMs.  The daos build for the doas RPMs should only build and package doas and no other components/dependencies.